ci-operator/config/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main.yaml

```
@@ -0,0 +1,38 @@
+base_images:
+  upi-installer:
+    name: "4.11"
+    namespace: ocp
+    tag: upi-installer
+binary_build_commands: make linux-amd64
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.11"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  upi-installer:
+    requests:
+      cpu: "3"
+      memory: 5Gi
+tests:
+- as: provider-certification-tool-vsphere
+  cron: 0 0 * * *
```

What about to run once a week at Sundays?
```suggestion
  cron: 0 0 * * 0
```